gadopt/stokes_integrators.py

```diff
@@ -130,17 +148,17 @@ def __init__(self, z, T, approximation, bcs=None, mu=1,
         self.strong_bcs = []
         for id, bc in bcs.items():
             weak_bc = {}
-            for type, value in bc.items():
-                if type == 'u':
+            for bc_type, value in bc.items():
```

Comment:
@stephankramer So I changed `type` to `bc_type` to avoid shadowing the built-in function. Mostly because my formatter was refusing to let it go! :-D

Reply:
Fair enough - I think you previously also had a `raise TypeError(f"Unexpected {type(bla)}")` in the same context which may also have caused issues - but yeah fine with that